app/models/user_session_form.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
```

Comment:
Why move this to models?

Reply:
it's a `FormObject` not a `Service`, all the tutorials recommend putting it in `app/models` or `app/forms` so just following those as a guide.

Reply:
Eh, ok. It doesn't seem like a model to me, but I see your point about it not being a service either.